### PR TITLE
[Keyboard]Update X-Bows Nature Keyboard

### PR DESCRIPTION
**Pull request by XBowsTech**
## Description

This is a minor update.
Update keymap.c and rules.mk.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
Got conflicts here that will need to be resolved before it can be merged, as well as what looks like a bunch of old commits.

File: keyboards/xbows/nature/config.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#define MATRIX_COLS 15
+#define MATRIX_ROW_PINS { F7, F6, F5, F4, F1, F0 }
+#define MATRIX_COL_PINS { B0, B1, B2, B3, B7, D2, D3, D5, D4, D6, D7, B4, B5, B6, C6 }
+#define DIODE_DIRECTION COL2ROW
+
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_MATRIX_LED_PROCESS_LIMIT 18
+#    define RGB_MATRIX_LED_FLUSH_LIMIT 16
+#    define DEBOUNCE 3
+#    define RGB_DISABLE_AFTER_TIMEOUT 0          // number of ticks to wait until disabling effects
```
Comment (drashna) [inline keyboards/xbows/nature/config.h:36]: I don't think you meant to put the debounce in the rgb matrix section? 

```suggestion
#define DEBOUNCE 3

#ifdef RGB_MATRIX_ENABLE
#    define RGB_MATRIX_LED_PROCESS_LIMIT 18
#    define RGB_MATRIX_LED_FLUSH_LIMIT 16
#    define RGB_DISABLE_AFTER_TIMEOUT 0          // number of ticks to wait until disabling effects
```

File: keyboards/xbows/nature/nature.c [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+ }, {
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4
+ } };
+
+#endif
```
Comment (drashna) [inline keyboards/xbows/nature/nature.c:138]:
```suggestion
```

File: keyboards/xbows/nature/nature.c [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
+
+ __attribute__ ((weak)) void rgb_matrix_indicators_user(void) {
+    if (host_keyboard_led_state().caps_lock) {
+        rgb_matrix_set_color(45, 0xFF, 0x00, 0x00);
+    }
+}
```
Comment (drashna) [inline keyboards/xbows/nature/nature.c:155]:
```suggestion
}

#endif
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
There are still merge conflicts here. 

You can rebase locally to fix this.  Or you can use the UI here to resolve the conflicts. 

If you need any help with doing so, just let us know.

**Comment by XBowsTech** [comment]
> There are still merge conflicts here.
> 
> You can rebase locally to fix this. Or you can use the UI here to resolve the conflicts.
> 
> If you need any help with doing so, just let us know.

OK, I will submit a new application.

**Pull request closed without merging**